src/Resources/Resources/ActiveDirectory/Cmdlets/UpdateAzureADApplicationCommand.cs

```diff
@@ -102,7 +102,7 @@ public override void ExecuteCmdlet()
                     Homepage = HomePage,
                     IdentifierUris = IdentifierUri,
                     ReplyUrls = ReplyUrl,
-                    AvailableToOtherTenants = AvailableToOtherTenants
+                    AvailableToOtherTenants = MyInvocation.BoundParameters.ContainsKey("AvailableToOtherTenants") ? AvailableToOtherTenants : (bool?)null
```

Comment:
Suggest to use IsParamaterBound to determine whether parameter is provided, just as line 90:
```
AvailableToOtherTenants = this.IsParameterBound(c => c.AvailableToOtherTenants ) ? AvailableToOtherTenants : (bool?)null;
```